core/src/main/java/cz/fi/muni/pv217/employee/management/service/OrderService.java

```diff
@@ -0,0 +1,72 @@
+package cz.fi.muni.pv217.employee.management.service;
+
+import cz.fi.muni.pv217.employee.management.entity.Order;
+import cz.fi.muni.pv217.employee.management.enums.OrderState;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.json.JsonObject;
+import javax.transaction.Transactional;
+import javax.ws.rs.NotFoundException;
+
+@ApplicationScoped
+public class OrderService {
+
+    @Transactional
+    public Order createOrder(Order order) {
+        order.persist();
+        return order;
+    }
+
+    @Transactional
+    public Order updateOrder(long id, JsonObject updateJson) {
```

Comment:
Why is this update method using JsonObject instead of Order? I think the data should be deserialized by now

Reply:
xstefank implement update method in example project like this. I don't know why. What do you suggest? How do we do it ?

Reply:
If it was shown as a Example there is surely a reason for it to be this way, so I am resolving this. I guess the reason is that you can specify only some fields and not send the whole entity.